static/app/components/links/link.tsx

```diff
@@ -68,17 +71,34 @@ const Link = withRouter(BaseLink);
 
 export default Link;
 
-const Anchor = styled('a', {
+const linkStyles = ({disabled, theme}: {theme: Theme; disabled?: boolean}) => `
+  border-radius: ${theme.borderRadius};
+
+  &.focus-visible {
+    box-shadow: ${theme.focusBorder} 0 0 0 2px;
+    text-decoration: none;
+    outline: none;
+  }
+
+  ${
+    disabled &&
+    `
+      color:${theme.disabled};
+      pointer-events: none;
+      :hover {
+        color: ${theme.disabled};
+      }
+    `
+  }
+`;
+
+const StyledRouterLink = styled(RouterLink)`
+  ${linkStyles}
+`;
+
+export const Anchor = styled('a', {
   shouldForwardProp: prop =>
     typeof prop === 'string' && isPropValid(prop) && prop !== 'disabled',
 })<{disabled?: boolean}>`
-  ${p =>
-    p.disabled &&
-    `
-  color:${p.theme.disabled};
-  pointer-events: none;
-  :hover {
-    color: ${p.theme.disabled};
-  }
-  `};
+  ${linkStyles}
```

Comment:
exporting this component along with its styles to be reused in `ExternalLink` and `LinkWithConfirmation`.